mixin/dashboards/rule.libsonnet

```
@@ -40,7 +40,7 @@ local utils = import '../lib/utils.libsonnet';
               (
                 max by(%(dimensions)s, rule_group) (prometheus_rule_group_last_duration_seconds{%(selector)s})
                 >
-                sum by(%(dimensions)s, rule_group) (prometheus_rule_group_interval_seconds{%(selector)s})
+                sum by(%(dimensions)s, rule_group) (prometheus_rule_group___rate_interval_seconds{%(selector)s})
```

I think this might not be intended?
```suggestion
                sum by(%(dimensions)s, rule_group) (prometheus_rule_group_interval_seconds{%(selector)s})
```

---

totally, will fix, thanks!